### DISCUSSION TITLE
Last update Time wrong

### DISCUSSION THREAD

**Discussion by knoellix** [nlef/moonraker-telegram-bot#296] [Q&A]
Is there a way to adjust the update time? I have the correct time zone on the system and the printer. but the message seems to use a different time zone.

![image](https://github.com/nlef/moonraker-telegram-bot/assets/128321164/ae772efc-be3f-4897-bafc-20b1285f10de)

**Comment by nlef** [comment]
@knoellix 
The time of the last update is taken relative to the date and time of the host on which the bot is running. Please check the date and time on the host

**Comment by knoellix** [comment]
I had actually already checked. but after resetting and restarting it now works properly.
